src/next/components/Snippet/Usercentrics.tsx

```diff
@@ -1,17 +1,25 @@
 import React, {ReactElement} from "react";
 import Script from "next/script";
+import {splitLocale} from "../../../lib/locale";
 
 export type UsercentricsProps = Readonly<{
 	id: string;
+	production?: boolean;
+	locale: string;
 }>;
 
 export function Usercentrics (props: UsercentricsProps): ReactElement | null
 {
+	const isProd = true === props.production;
+	const language = splitLocale(props.locale).at(0) ?? "en";
```

Comment:
We should instead omit it as default, as then they use the browser language IIRC